lib/util.js

```diff
@@ -227,6 +227,18 @@ function encode_uri(uri) {
 exports.encode_uri = encode_uri;
 
 function fetch_rss_entries (feed, max_items=Infinity, max_days_old=1) {
+    // Allow environment variables to override this
+    try {
+        if (process.env.LIBINGESTER_MAX_ITEMS) {
+            max_items = parseInt(process.env.LIBINGESTER_MAX_ITEMS);
+        }
+        if (process.env.LIBINGESTER_MAX_DAYS_OLD) {
+            max_days_old = parseInt(process.env.LIBINGESTER_MAX_DAYS_OLD);
+        }
+    } catch (err) {
+        throw new Error(`Couldn't parse RSS entry overrides: ${err}`);
+    }
```

Comment:
I think you can do this without try/catch:
```
node
> host = "blah"
'blah'
> process.env['MISSING']
undefined
> process.env['MISSING'] || host
'blah'
> process.env['HOST'] || host
'localhost
```

Reply:
Edit: maybe not the parsing I suppse

Reply:
yeah, the parsing was why I wrapped them